author/topics/document-format/collections.adoc

```diff
@@ -199,6 +190,88 @@ final-content:
   Hic explicit
 ----
 
+=== Site manifest
```

Comment:
Thanks a lot for adding this section. I'm thinking out loud - should this section belong to collection documentation or site generation documentation? and how does this collection manifesto is different compare to the explanation we have for [collection manifesto here](https://www.metanorma.org/author/topics/document-format/collections/#collection-manifest)?

Reply:
It belongs here, because it is how to use sites in the context of collections: it's about how files are listed in both. The explanation is only about that, and it does not go into all the other things discussed in the collection manifest above.